package.js

```diff
@@ -19,6 +19,15 @@ Package.onUse(function(api) {
   // font files
   var fontDirectory = 'bower_components/materialize/';
   var fontFiles = [
+    'font/material-design-icons/Material-Design-Icons.eot',
```

Comment:
Material-Design-Icons are not used and against materializecss doc. We should not include them any more.